cmd/ui/src/ducks/explore/saga.ts

```diff
@@ -195,15 +195,10 @@ function* runCypherSearchQuery(payload: CypherQueryRequest): SagaIterator {
 
         if (resultNodesAreEmpty && !resultEdgesAreEmpty) {
             yield put(putGraphData({}));
-            yield put(
-                addSnackbar(
-                    'The results are not rendered since only edges were returned',
-                    'cypherSearchOnlyContainsEdges'
-                )
-            );
+            yield put(addSnackbar('The results are not rendered since only edges were returned', 'cypherSearchOnlyContainsEdges'));
         } else if (resultNodesAreEmpty && resultEdgesAreEmpty) {
             yield put(putGraphData({}));
-            yield put(addSnackbar('No results match your criteria', 'cypherSearchEmptyResponse'));
+            yield put(addSnackbar('Command completed successfully', 'cypherSuccessResponse'));
```

Comment:
Should we also add this new message in the below else block? For queries that do not error and have results.

Reply:
If they have results we just display the results with no snackbar like previously.

Reply:
Makes sense! Just was double checking on the AC :) Thank you!